website/api/bonsai/verifier-contract/overview.md

```diff
@@ -0,0 +1,43 @@
+# Verifier Contract
+
+To utilize the results generated by a [zkVM program][term-zkvm-program] in the Blockchain, it’s essential to confirm their correctness and integrity. This [verification][term-verify] is accomplished through a verifier contract. Specifically, RISC Zero’s verifier contract performs the task of validating [Groth16 SNARK][article-groth16] proofs, which are produced by the [Bonsai proving service][Bonsai].
+
+The verifier contract incorporates the [`IRiscZeroVerifier` interface][IRiscZeroVerifier.sol], which defines the necessary data structures for the receipt and includes a `verify` method. This interface is implemented by the [`RiscZeroGroth16Verifier` contract][RiscZeroGroth16Verifier.sol], a stateless and immutable verifier designed for RISC Zero Groth16 SNARK proofs.
+
+Given the nature of the zkVM, any computation performed off-chain can be proven on-chain using a single verifier contract. This simplifies system architecture and eliminates the need for multiple contracts for different circuits. Even in the case of a system that needs to use custom circuits, it's possible to prove the resulting custom [receipts][term-receipt] within a [zkVM program][term-zkvm-program] and submit the resulting proof to the verifier contract.
+
+## Interface
+
+<!-- TODO: Link to auto-generated Solidity annotation docs -->
+
+TODO
```

Comment:
Just not sure which content link here yet - I think it's in Victor's local branch?

Reply:
I'll add this in a follow-up PR. I've removed this heading for now.